src/python/platforms/android/gestures.py

```diff
@@ -22,28 +22,26 @@
 MONKEY_THROTTLE_DELAY = 100
 
 # Maximum number of monkey events per testcase.
-NUM_MONKEY_EVENTS = 50
+NUM_MONKEY_EVENTS = 25
 
 
-def get_random_gestures(gesture_count):  # pylint: disable=unused-argument
+def get_random_gestures(_):
   """Return a random gesture seed from monkey framework natively supported by
   Android OS."""
   random_seed = random.getrandbits(32)
   gesture = 'monkey,%s' % str(random_seed)
   return [gesture]
 
 
-# pylint: disable=unused-argument
-def run_gestures(gestures, start_time, timeout):
+def run_gestures(gestures, *_):
   """Run the provided interaction gestures."""
   package_name = os.getenv('PKG_NAME')
   if not package_name:
     # No package to send gestures to, bail out.
     return
 
-  if len(gestures) != 1 or not gestures[0].startswith('monkey'):
-    # Bad gesture string, bail out.
-    return
+  assert len(gestures) == 1
+  assert gestures[0].startswith('monkey,')
```

Comment:
Since we accept user-supplied strings this could lead to exceptions rather than errors if someone ever types something random in for Android. The old behavior seems slightly better to me.

Reply:
Makes sense. Reverting.